pravega/bookkeeper-operator
# Issue119: add bk label support

### Change log description

add  Labels crd

### Purpose of the change

Fixes #119

### What the code does

_(Detailed description of the code changes)_

### How to verify it

Deploy bookkeeper with customer labels set, use kubetcl describe pod to see if label succesfully attached to bookie pods
